File: src/functions/payment-net/payment/Payment.cs
public class Payment
{
    public decimal Amount { get; set; }
    public string Currency { get; set; }
    public Card Card { get; set; }
}

public class Card
{
    public string Number { get; set; }
    public string Expiry { get; set; }
    public string CVV { get; set; }
}

public class PaymentRequest
{
    public Payment Payment { get; set; }
}

File: src/az-functions/payment-py/payment.py
"""Payment model for payment processing."""

from typing import Optional
from decimal import Decimal
from card import Card


class Payment:
    """Represents a payment with amount, currency, and card information."""
    
    def __init__(self, amount: Decimal = Decimal('0.00'), currency: str = "USD", card: Optional[Card] = None):
        self.amount = amount
        self.currency = currency
        self.card = card or Card()
    
    def to_dict(self) -> dict:
        """Convert Payment object to dictionary."""
        return {
            "amount": str(self.amount),
            "currency": self.currency,
            "card": self.card.to_dict()
        }
    
    @classmethod
    def from_dict(cls, data: dict) -> 'Payment':
        """Create Payment object from dictionary."""
        amount = Decimal(str(data.get("amount", "0.00")))
        currency = data.get("currency", "USD")
        card_data = data.get("card", {})
        card = Card.from_dict(card_data) if card_data else Card()
        
        return cls(amount=amount, currency=currency, card=card)
